modular_zzplurt/code/modules/mob/living/silicon/robot/robot_defines.dm

```diff
@@ -0,0 +1,3 @@
+/mob/living/silicon/robot
+	maxHealth = 120
+	health = 120
```

Comment:
You could (Read: consider this and maybe discuss with others) use HUMAN_MAXHEALTH with these to always keep borgos consistent with future changes.